test/tests/aws_instance_volume_tags/expected/main.tf

```diff
@@ -16,14 +16,27 @@ resource "aws_instance" "ubuntu" {
   instance_type     = "t3.micro"
   availability_zone = "us-west-2"
 
-
   tags = merge({
     "Name" = "terratag-test"
     "env"  = "test"
   }, local.terratag_added_main)
   volume_tags = local.terratag_added_main
 }
 
+resource "aws_instance" "ubuntu2" {
+  ami               = "dasdasD"
+  instance_type     = "t3.micro"
+  availability_zone = "us-west-2"
+
+  root_block_device {
```

Comment:
Can you please also add the volume_tags block here and make sure it's not altered?

Reply:
You can't... it will create a confilct.
volume_tags should have been added, but wasn't added because root_block_device exist.

Reply:
@yaronya - based on the documentation, I've made larger changes. (it's less straighforward than I initially assumed).
Please recheck.

See the tag guide:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/instance#tag-guide